frontend/src/lib/utils.test.ts

```diff
@@ -334,6 +335,85 @@ describe('dateFilterToText()', () => {
     })
 })
 
+describe('dateStringToDayJs', () => {
+    beforeEach(() => {
+        tk.freeze(1330688329321)
```

Comment:
What date/time is this?

Reply:
```ts
        tk.freeze(1330688329321) // randomly chosen time on the 22nd of February 2022
```

Should it have meaning?

Reply:
Just harder to "imagine" the correct result in tests without knowing what time this magic constant represents 😅